src/main/groovy/com/bmuschko/gradle/docker/DockerRemoteApiPlugin.groovy

```diff
@@ -47,14 +47,14 @@ class DockerRemoteApiPlugin implements Plugin<Project> {
                 project.repositories.addAll(project.buildscript.repositories.collect())
             }
         }
-        
```

Comment:
No, my IDE automatically removes trailing spaces. I can revert it in the pull request for the version update if it disturbs.